### PR TITLE
Update version: FreeCAD.FreeCAD version 0.20.2

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/114470&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/f/FreeCAD/FreeCAD/0.20.2/FreeCAD.FreeCAD.installer.yaml [hunk 1]
```diff
@@ -113,22 +113,19 @@ FileExtensions:
...
-  InstallerUrl: https://github.com/FreeCAD/FreeCAD/releases/download/0.20.2/FreeCAD-0.20.2-WIN-x64-installer-3.exe
-  InstallerSha256: 3417D7079810E41DC2FF3C61FD2BCC0B121612E62CB92D3DD7F1E6935F938B74
-  InstallerSwitches:
-    Custom: /ALLUSERS
-- Architecture: x64
-  InstallerType: nullsoft
   Scope: user
-  InstallerUrl: https://github.com/FreeCAD/FreeCAD/releases/download/0.20.2/FreeCAD-0.20.2-WIN-x64-installer-3.exe
-  InstallerSha256: 3417D7079810E41DC2FF3C61FD2BCC0B121612E62CB92D3DD7F1E6935F938B74
+  InstallerUrl: https://github.com/FreeCAD/FreeCAD/releases/download/0.20.2/FreeCAD-0.20.2-WIN-x64-installer-2.exe
```
Comment (russellbanks) [inline manifests/f/FreeCAD/FreeCAD/0.20.2/FreeCAD.FreeCAD.installer.yaml:120]: Wouldn't installer-3 be newer?

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
This package installed normally, but gave an error when launched:
> Faulting application name: FreeCAD.exe, version: 0.20.2.2, time stamp: 0x640cf95b
> Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000
> Exception code: 0xc0000005
> Fault offset: 0x0000000000000000
> Faulting process id: 0x500
> Faulting application start time: 0x01d9ca4dcc8c4b1b
> Faulting application path: C:\Users\User\AppData\Local\Programs\FreeCAD 0.20\bin\FreeCAD.exe
> Faulting module path: unknown
Report Id: 710acf16-a275-4ed0-bcfc-aff4b96c6081
> Faulting package full name: 
> Faulting package-relative application ID:

File: manifests/f/FreeCAD/FreeCAD/0.20.2/FreeCAD.FreeCAD.installer.yaml [hunk 2]
```diff
@@ -113,22 +113,19 @@ FileExtensions:
 - yaml
 - z88
 - zip
+ReleaseDate: 2023-01-07
 Installers:
 - Architecture: x64
-  InstallerType: nullsoft
-  Scope: machine
+  Scope: user
```
Comment (stephengillie) [inline manifests/f/FreeCAD/FreeCAD/0.20.2/FreeCAD.FreeCAD.installer.yaml:119]:
```suggestion
  Scope: machine
```

File: manifests/f/FreeCAD/FreeCAD/0.20.2/FreeCAD.FreeCAD.installer.yaml [hunk 3]
```diff
@@ -113,22 +113,19 @@ FileExtensions:
...
 - zip
+ReleaseDate: 2023-01-07
 Installers:
 - Architecture: x64
-  InstallerType: nullsoft
-  Scope: machine
+  Scope: user
   InstallerUrl: https://github.com/FreeCAD/FreeCAD/releases/download/0.20.2/FreeCAD-0.20.2-WIN-x64-installer-3.exe
-  InstallerSha256: 3417D7079810E41DC2FF3C61FD2BCC0B121612E62CB92D3DD7F1E6935F938B74
+  InstallerSha256: 3417d7079810e41dc2ff3c61fd2bcc0b121612e62cb92d3dd7f1e6935f938b74
```
Comment (stephengillie) [inline manifests/f/FreeCAD/FreeCAD/0.20.2/FreeCAD.FreeCAD.installer.yaml:121]:
```suggestion
  InstallerSha256: 3417D7079810E41DC2FF3C61FD2BCC0B121612E62CB92D3DD7F1E6935F938B74
```

File: manifests/f/FreeCAD/FreeCAD/0.20.2/FreeCAD.FreeCAD.installer.yaml [hunk 4]
```diff
@@ -113,22 +113,19 @@ FileExtensions:
...
   InstallerSwitches:
-    Custom: /ALLUSERS
+    Custom: /CURRENTUSER
```
Comment (stephengillie) [inline manifests/f/FreeCAD/FreeCAD/0.20.2/FreeCAD.FreeCAD.installer.yaml:123]:
```suggestion
    Custom: /ALLUSERS
```

File: manifests/f/FreeCAD/FreeCAD/0.20.2/FreeCAD.FreeCAD.installer.yaml [hunk 5]
```diff
@@ -113,22 +113,19 @@ FileExtensions:
...
 - Architecture: x64
-  InstallerType: nullsoft
-  Scope: user
+  Scope: machine
   InstallerUrl: https://github.com/FreeCAD/FreeCAD/releases/download/0.20.2/FreeCAD-0.20.2-WIN-x64-installer-3.exe
-  InstallerSha256: 3417D7079810E41DC2FF3C61FD2BCC0B121612E62CB92D3DD7F1E6935F938B74
+  InstallerSha256: 3417d7079810e41dc2ff3c61fd2bcc0b121612e62cb92d3dd7f1e6935f938b74
   InstallerSwitches:
-    Custom: /CURRENTUSER
+    Custom: /ALLUSERS
```
Comment (stephengillie) [inline manifests/f/FreeCAD/FreeCAD/0.20.2/FreeCAD.FreeCAD.installer.yaml:129]:
```suggestion
    Custom: /CURRENTUSER
```

**Review by stephengillie — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**